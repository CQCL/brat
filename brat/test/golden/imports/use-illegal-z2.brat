import lib (x, y)

main :: Nat, Int, Bool
main = lib.x, lib.y, lib.z
