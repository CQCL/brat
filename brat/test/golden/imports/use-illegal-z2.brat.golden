Error in test/golden/imports/use-illegal-z2.brat@4:22-27
main = lib.x, lib.y, lib.z
                     ^^^^^

  lib.z not found in (value) environment

